src/Service/ReminderStatus.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Service;
+
+abstract class ReminderStatus
+{
+    public const Pending = 1;
+    public const Active = 2;
+    public const Exceeded = 3;
```

Comment:
Constants are usually written in all caps.

Reply:
Used caps for constants.